packages/main/src/Button.js

```diff
@@ -315,6 +327,26 @@ class Button extends UI5Element {
 		};
 	}
 
+	get ariaLabelledByText() {
+		if (!this.ariaLabelledby) {
+			return undefined;
+		}
+
+		const ids = this.ariaLabelledby.split(" ");
+		let result = "";
+
+		ids.forEach((elementId, index) => {
+			const element = document.querySelector(`#${elementId}`);
```

Comment:
This should not be the document, but the HTML document of the ui5-button itself. It may be an iframe, a shadow root, etc...
Otherwise when you have ui5-button in a shadow root for example this won't work.

Reply:
Done

Reply:
This line should be: ```const element = owner.querySelector(`[id='${elementId}']`);```
Otherwise it causes this error ```DOMException: Failed to execute 'querySelector' on 'Element':``` for some id's

Reply:
Fixed in #1500